internal/quaiapi/quai_api.go

```diff
@@ -497,15 +497,12 @@ func (s *PublicBlockChainQuaiAPI) rpcMarshalBlock(ctx context.Context, b *types.
 	if err != nil {
 		return nil, err
 	}
-	// return order of the block in the api
-	order, err := b.Header().CalcOrder()
+	_, order, err := s.b.CalcOrder(b.Header())
 	if err != nil {
 		return nil, err
 	}
 	fields["order"] = order
-	if inclTx {
```

Comment:
Why is inclTx no longer needed?

Reply:
This was a bugfix that I accidentally squashed in. I can remove it to a separate bugfix if you like.

The `totalEntropy` field _should_ be marshalled even if the block is not marshalling any transactions.

Reply:
Great, no I think we can leave it as is. Just wanted to understand the change as it seemed out of place.